build-tests/sparo-outputs/config/heft.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/v0/heft.schema.json",
+
+  "extends": "@rushstack/heft-node-rig/profiles/default/config/heft.json",
+
+  "phasesByName": {
+    "build": {
+      "tasksByName": {
+        "post-compile": {
+          "taskDependencies": ["typescript"],
+
+          "taskPlugin": {
+            "pluginName": "run-script-plugin",
+            "pluginPackage": "@rushstack/heft",
+
+            "options": {
+              "scriptPath": "lib/generateOutputs.js"
```

Comment:
Entry points should be clearly indicated by their filename (something like `lib/start.js` or maybe `lib/start-test.js`)